test/VSIXBootstrapper.Test/stdafx.h

```diff
@@ -14,3 +14,10 @@
 #include "CppUnitTest.h"
 
 // Project header files
+
+// Disable Prefast warnings
+#pragma warning(disable: 6101) // Returning uninitialied data. This is from test cases that fail if the code executes.
+#pragma warning(disable: 6054) // Buffer may not be null-terminated.
+#pragma warning(disable: 6385) // Reading extra data
+#pragma warning(disable: 6386) // Potential buffer overrun.
```

Comment:
This one in particular is a bit frightening to see suppressed globally. Are you sure?

Reply:
Yeah let me find which test was flagging it and take a look. Would you prefer individual suppression on these? For the tests I took a broader approach to avoid doing too much work, a lot was flagged for intentional fail cases and fall-throughs

Reply:
That's a good point and I admit I missed that: these are for tests. In that case, I don't particularly care, except maybe that 6386 one. It would be good to at least track that down and see why it was raised. It could potentially show fault in shipping code if something isn't bounds-checked properly. Maybe even the 6054. If it's just in test code, I think what you have then is fine. I'll unblock if that's the case.

Reply:
I just decided to suppress these individually where needed